src/form/src/components/Form.js

```diff
@@ -41,23 +42,37 @@ const HiddenButton = styled.button`
 
 type Props = {
   disabled :boolean;
-  onSubmit :() => void;
+  hideSubmit :boolean;
+  isSubmitting :boolean;
   onChange :() => void;
-  onDiscard ? :() => void;
-  isSubmitting ? :boolean;
+  onDiscard :() => void;
+  onSubmit :() => void;
+  ref :ElementRef<typeof StyledForm>;
```

Comment:
How about `submitRef` and also `discardRef`?

Reply:
AFAIK you can only forwardRef one ref at a time. 

Since the discard button is opt in and none of its functionality is dictated by RJSF or any semantic html regarding forms, it doesn't make much sense to make the ref available. 

Having an external element call `onDiscard` would be identical and less convoluted than using a ref. Exposing the form ref and triggering the 'submit' event is the same as clicking on the `type='submit'` button.